sdks/python/apache_beam/transforms/sql_test.py

```diff
@@ -157,6 +157,21 @@ def test_zetasql_generate_data(self):
           dialect="zetasql")
       assert_that(out, equal_to([(1, "foo", 3.14)]))
 
+  def test_windowing_before_sql(self):
```

Comment:
Do you think we could come up with a test where PTransforms get assigned to wrong IDs during pipeline expansion (resulting in an invalid pipeline proto) ?